fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -1512,432 +1526,477 @@ private Bundle processBundleEntries(Bundle requestBundle, Bundle responseBundle,
-                            // Add warning and hint issues to response outcome if any.
-                            if (result instanceof OperationOutcome) {
-                                if (((OperationOutcome) result).getIssue() != null) {
-                                    responseBuilder.outcome(result);
-                                }
-                            }
-
-                            responseBuilder.status(string(Integer.toString(SC_OK)));
-                            responseIndexAndEntries.put(entryIndex, responseEntryBuilder
-                                    .resource(result)
-                                    .response(responseBuilder.build())
-                                    .build());
-                            setBundleResponseStatus(response, SC_OK, requestDescription.toString(), initialTime);
-
-                        } else if (pathTokens.length == 2 && "_search".equals(pathTokens[1])) {
-                            // This is a 'search' request.
-                            Bundle searchResults =
-                                    doSearch(pathTokens[0], null, null, queryParams, absoluteUri, null, null);
-
-                            // Save the results of the operation in the bundle response field.
-                            Bundle.Entry.Response.Builder responseBuilder = response.toBuilder();
-                            responseBuilder.status(string(Integer.toString(SC_OK)));
-
-                            responseIndexAndEntries.put(entryIndex, responseEntryBuilder
-                                    .resource(searchResults)
-                                    .response(responseBuilder.build())
-                                    .build());
-
-                            setBundleResponseStatus(response, SC_OK, requestDescription.toString(), initialTime);
-                        } else if (pathTokens.length == 1) {
-                            // This is a 'create' request.
-
-                            // Retrieve the local identifier from the request entry (if present).
-                            String localIdentifier =
-                                    retrieveLocalIdentifier(requestEntry, localRefMap);
-
-                            // Retrieve the resource from the request entry.
-                            Resource resource = requestEntry.getResource();
-                            if (resource == null) {
-                                String msg =
-                                        "BundleEntry.resource is required for bundled create requests.";
-                                throw buildRestException(msg, IssueType.NOT_FOUND);
-                            }
-
-                            // Convert any local references found within the resource to their
-                            // corresponding external reference.
+                        responseBuilder.status(string(Integer.toString(SC_NOT_FOUND)));
```

Comment:
since this if fixed, it may be great to make that string value a static final `string(Integer.toString(SC_NOT_FOUND))`

Reply:
Done.